devicemodel/hw/pci/core.c

```diff
@@ -750,9 +750,10 @@ pci_emul_alloc_pbar(struct pci_vdev *pdi, int idx, uint64_t hostbase,
 		    enum pcibar_type type, uint64_t size)
 {
 	int error;
-	uint64_t *baseptr, limit, addr, mask, lobits, bar;
+	uint64_t *baseptr, limit = 0, addr, mask, lobits, bar;
```

Comment:
uint64_t *baseptr = NULL, limit = 0, addr, mask, lobits, bar;

Reply:
update the code, thanks for your review.